### PR TITLE
FIxes apiary text for queen placement.

### LINKED ISSUE #15953: Minor text change required for apiary
**Issue Description**:

Apiaries. say this when you place a queen into the box

"sets the queen bee down inside the apiary, making it his new home."

**What did you expect to happen**: 

Queens, as the title implies... are females insects.

**What happened instead**:

It should say HER instead

**Why is this bad/What are the consequences**:

Self explanatory...

**Steps to reproduce the problem**:

N/A

**When did the problem start happening**:

N/A

**Extra information**:

### PR DESCRIPTION
**Pull request by Dragonkiller93**
## What Does This PR Do
Changes apiary queen placement text to read "making it her new home" since queens are female bees. Fixes #15953 

## Why It's Good For The Game
Consistency in the descriptions of actions is good to have. Kind of annoying seeing minor issues like this, might as well fix them.

## Changelog
:cl: Iwanabeu
tweak: Apiary text upon placing a queen bee now uses her instead of his, since queens are female.
/:cl:

### CODE REVIEW

File: code/modules/hydroponics/beekeeping/beebox.dm [hunk 1]
```diff
@@ -178,7 +178,7 @@
 			return
 
 		if(queen_bee)
-			visible_message("<span class='notice'>[user] sets [qb] down inside the apiary, making it [user.p_their()] new home.</span>")
+			visible_message("<span class='notice'>[user] sets [qb] down inside the apiary, making it her new home.</span>")
```
Comment (farie82) [inline code/modules/hydroponics/beekeeping/beebox.dm:181]:
```suggestion
			visible_message("<span class='notice'>[user] sets [qb] down inside the apiary, making it her [qb.p_their()] home.</span>")
```

**Review by farie82 — CHANGES_REQUESTED** [review]
Every atom has a `gender` variable. The default is `NEUTER`. Changing that will properly change the her/his/their thing.

**Comment by Dragonkiller93** [comment]
Why replace "new" with the verb?

**Comment by Dragonkiller93** [comment]
This github stuff has got me real confused, I think the right changes happened in the end?

**Review by farie82 — APPROVED** [review]
> This github stuff has got me real confused, I think the right changes happened in the end?

Looks good to me!

**Review by S34NW — APPROVED** [review]

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**